add login modal window in hall page, add 15 min timer

kds1660 opened the pull request:
1. add timer directive/controller, when user add place to cart timer
start (15 minutes), when user empty cart - stop timer , add styles for timer
2. translate for login modal window, timer
3. for social add social button angular module with fonts
3. add modal login page with facebook for now login (send my app id for now, must change) and send login request when modal login window close, else if user dont login

kds1660 committed:
Merge remote-tracking branch 'refs/remotes/origin/dev' into Geekhub-2014-JS/dev

kds1660 committed:
Merge remote-tracking branch 'refs/remotes/origin/dev' into Geekhub-2014-JS/dev

kds1660 committed:
add facebook login page

1. add login controller
2. for login page add  bootstrap-social and font-awesome
3. change hall controller - add login modal at load page

kds1660 committed:
changes

1. autoclose login modal page after login
2. in hallctrl remove click event fot places with class bougt and
reserved

kds1660 committed:
Merge remote-tracking branch 'refs/remotes/Geekhub-2014-JS/dev' into dev

kds1660 committed:
add login and timer

1. add timer directive/controller, when user add place to cart timer
start (15 minutes), add styles for timer
2. translate for login modal window
3. add send request when modal login window close, else if user dont
login
4. change user from rootscope to userService

Varkh commented on src/css/hall.css:
```diff
@@ -249,3 +249,13 @@ li {
+
+.modal-content {
+  background: #ffcdcd;
+  !important;
```
what is `!important` here?

kds1660 replied:
owerwrite standart bootstrap modal white background, without bootstrap css priority highter

Varkh commented on src/js/Controllers/LoginCtrl.js:
```diff
@@ -0,0 +1,52 @@
+            firstScriptElement.parentNode.insertBefore(facebookJS, firstScriptElement);
+        }());
+    })
+    .controller('LoginCtrl', ['$scope', '$modalInstance','$rootScope', '$facebook', 'apiPost', 'userService',
```
move to different file: separate config and controller

kds1660 replied:
in site config in module file,  I'm sticking to the logic of the site

Varkh commented on src/js/Controllers/LoginCtrl.js:
```diff
@@ -0,0 +1,52 @@
+                            if (response.status === 'connected') {
+                                userService.setApiToken(response.authResponse.accessToken);
+                                userService.setNetwork('facebook');
+                                /*$rootScope.user.accessToken = response.authResponse.accessToken;
```
no comments in code. remove

Varkh commented on src/js/Controllers/LoginCtrl.js:
```diff
@@ -0,0 +1,52 @@
+                                /*$rootScope.user.accessToken = response.authResponse.accessToken;
+                                $rootScope.user.network = 'facebook';*/
+                            } else {
+                                // the user isn't logged in to Facebook.
```
what should be here?

kds1660 replied:
backend give me api token, thay dont implemented this yet...

kds1660 replied:
i will remove this and call api get request, when modal window closed

Varkh commented on src/js/Controllers/hallCtrl.js:
```diff
@@ -26,6 +27,75 @@ angular.module('hall', [])
+            }).result.finally(function(){
+                //if (!$rootScope.user) $rootScope.user={}; //Todo change to object
+                apiPost('customers/login',{
+                    "first_name": userService.getCurrentUser().first_name, //Todo to service
```
You can create ticket for all this TODOs, just don't left them unhandled

kds1660 replied:
I solved them, forget remove

Varkh commented on src/js/Controllers/hallCtrl.js:
```diff
@@ -26,6 +27,75 @@ angular.module('hall', [])
+                $interval.cancel(timerInterval);
+            });
+
+            function getTimeRemaining(endtime){
```
Move to some Util factory

kds1660 replied:
Was going to do - timer service for hall and for shopping cart, now remake

kds1660 committed:
change request

also change venue variable from string to object (backend changed)

kds1660 committed:
hall ctrl changes

kds1660 committed:
add ticket api modelling

kds1660 committed:
tickets - compare changes from backend and update every second

kds1660 committed:
del console log

kds1660 committed:
fix ticket update

kds1660 committed:
separate module configs in config directory

kds1660 committed:
big commit

1.  add sum for cart
2.  add ticket registration page - add directive, timer send from hall
page and user-perfomance add from services
3. change css from frontend - hall is now  responsive
4. add in servise and change in directive - sit section and sit cost

kds1660 committed:
change timer view

kds1660 committed:
changes

kds1660 committed:
timer bug fix

kds1660 committed:
timer fix
